lib/testing/src/lib/process-services-cloud/pages/task-list-cloud-component.page.ts

```diff
@@ -39,120 +39,119 @@ export class TaskListCloudComponentPage {
 
     dataTable = new DataTableComponentPage(this.taskList);
 
-    getDataTable() {
+    getDataTable(): DataTableComponentPage {
         return this.dataTable;
     }
 
-    clickCheckbox(taskName) {
+    clickCheckbox(taskName): Promise<void> {
```

Comment:
needs async ?

Reply:
nope async needs to wrap te return in a promise this return already return a promise